src/web/handlers.rs

```diff
@@ -65,23 +28,45 @@ pub async fn get_tiles(
             .expect("Invalid ADM_COUNTRY_IP_MAP setting")
     };
     let stripped_ua = user_agent::strip_ua(&treq.ua);
+    let header = request.head();
+    let c_info = request.connection_info();
+    let ip_addr_str = c_info.remote_addr().unwrap_or(fake_ip);
```

Comment:
using the `fake_ip` here is dubious but I'm not sure what we should do otherwise.

Reply:
Let's keep it as is and just log an issue for it. If we go with the LB headers to start with we also need a similar "what should we do" when the LB returns an empty string for country/region.